scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ClientHandshaker.java

```diff
@@ -471,6 +471,7 @@ private void receivedServerHelloDone(ServerHelloDone message) throws HandshakeEx
 			generateKeys(premasterSecret);
 			break;
 		case PSK:
+			recordLayer.pauseRetransmission();
```

Comment:
You seems to say, that pause/resume is not necessary for client side ? why just for client side ?
You propose to just stop retransmission when message is received, right ?

I could be agree with that but this is no so easy :/
If we chose this solution,  where could be the right place to stop the retransmission ?
1) As soon as we received a new flight from the peer ?
[The spec](https://tools.ietf.org/html/rfc6347#section-4.1) seems to base retransmission on flight but we need to be able to detect duplicate, I'm not sure this really well match with our current implementation...
2) [As soon as we received a handshake message](https://github.com/eclipse/californium/blob/master/scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java#L842) ? 
This is not the right place, because message can be received in bad order or duplicate.
3) As soon as we receive the expected message (in Handshaker.doProcess() method) ?
Here we know we process the right message based on message sequence number, but there is no check about the nature of the message. I mean we can receive a message with the good sequence number but with bad type as [we doesn't store state](https://github.com/eclipse/californium/pull/309#issuecomment-304688301). 
More than that,  some time we don't expect one message but several messages (which could be sent in one flight or more), Should we ensure we received all messages before to stop retransmission ? or we just consider we can stop it when we receive the first message ?